Description: analog-input-front-mic add Jack Front Line Out Front
Author: <songwentai@uniontech.com>

Origin: https://gerrit.uniontech.com/plugins/gitiles/base/pulseaudio/+/a93d016acb670fe160b71baae7f282a41911189e
Last-Update: 2022-05-25

--- pulseaudio-15.0+dfsg1.orig/src/modules/alsa/mixer/paths/analog-input-front-mic.conf
+++ pulseaudio-15.0+dfsg1/src/modules/alsa/mixer/paths/analog-input-front-mic.conf
@@ -29,6 +29,9 @@ required-any = any
 state.plugged = unknown
 state.unplugged = unknown
 
+[Jack Front Line Out Front]
+required-any = any
+
 [Element Capture]
 switch = mute
 volume = merge
